cdist/conf/type/__apt_default_release/manifest

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh -e
+#
+# 2014 Steven Armstrong (steven-cdist at armstrong.cc)
+# 2017 Matthijs Kooijman (matthijs at stdin.nl)
```

Comment:
Did Steven participate in this type? If no then above lines are copy-paste relict :)

Reply:
Well, a lot of the code is a copy-paste from `__apt_source` IIRC, so copyright on those lines is not mine to claim. It's not really big parts, so removing his copyright notice would probably be fine too, if that's what you prefer.

Reply:
@matthijskooijman Ok, you can leave this as is.